hil-test/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+#![no_std]
+
+use defmt_rtt as _;
+use panic_probe as _;
+
+pub mod esp_hal {
```

Comment:
What's the purpose of this? Could we not just use `pub use esp_hal` here?

Reply:
Ups its a [leftover](https://github.com/esp-rs/esp-hal/pull/1297/commits/1ec70a8bb0c25d7dda2afa7b41cbde9bf697eed0#diff-8ca170be4c58f4bdbe4e779de8359cbe40188475a67baa692f3f84bcd3d9c46fR7-R20) of when we had several crates for hal, I quickly changed it to test it and forgot to update it. Will update it!